### PR TITLE
*: client vendoring README

### PR DESCRIPTION
**Pull request by gyuho**
If I understand our new vendoring correctly... please review /cc @xiang90 @heyitsanthony

### CODE REVIEW

File: client/README.md [hunk 1]
```diff
@@ -4,9 +4,10 @@ etcd/client is the Go client library for etcd.
 
 [![GoDoc](https://godoc.org/github.com/coreos/etcd/client?status.png)](https://godoc.org/github.com/coreos/etcd/client)
 
-etcd uses go's `vendor` directory to manage external dependencies. If `client` is imported
-outside of etcd, simply copy `client` to the `vendor` directory or use tools like godep to
-manage your own dependency, as in [vendor directories](https://golang.org/cmd/go/#hdr-Vendor_Directories).
+etcd uses `cmd/vendor` directory to store external dependencies, which are to be compiled into
+etcd release binaries. `client` can be imported without vendoring. For full compatibility, it
+is recommended to have the same dependency as etcd, using tools like godep, as in
```
Comment (heyitsanthony) [inline client/README.md]: `it is recommended to vendor builds using etcd's vendored packages`

**Comment by gyuho** [comment]
@heyitsanthony fixed. PTLA!

**Comment by heyitsanthony** [comment]
@gyuho please use my wording?

**Comment by gyuho** [comment]
@heyitsanthony Oh pushed the wrong commit. Should be fixed now? Thanks!

File: client/README.md [hunk 2]
```diff
@@ -4,9 +4,11 @@ etcd/client is the Go client library for etcd.
 
 [![GoDoc](https://godoc.org/github.com/coreos/etcd/client?status.png)](https://godoc.org/github.com/coreos/etcd/client)
 
-etcd uses go's `vendor` directory to manage external dependencies. If `client` is imported
-outside of etcd, simply copy `client` to the `vendor` directory or use tools like godep to
-manage your own dependency, as in [vendor directories](https://golang.org/cmd/go/#hdr-Vendor_Directories).
+etcd uses `cmd/vendor` directory to store external dependencies, which are
+to be compiled into etcd release binaries. `client` can be imported without
+vendoring. For full compatibility, it is recommended to vendor builds using
+etcd's vendored dependency, using tools like godep, as in
```
Comment (heyitsanthony) [inline client/README.md]: `etcd's vendored packages`

**Comment by gyuho** [comment]
fixed up. ptla. thanks!

**Comment by heyitsanthony** [comment]
lgtm defer to @xiang90

**Comment by xiang90** [comment]
lgtm

**Pull request merged**